include/envoy/ext_authz/ext_authz.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "envoy/common/optional.h"
+#include "envoy/common/pure.h"
+#include "envoy/http/filter.h"
+#include "envoy/http/header_map.h"
+#include "envoy/network/filter.h"
+#include "envoy/tracing/http_tracer.h"
+
+#include "api/auth/external_auth.pb.h"
+
+namespace Envoy {
+namespace ExtAuthz {
+
+using envoy::api::v2::auth::CheckRequest;
```

Comment:
`using` isn't allowed in headers by the style guide. It's also more readable IMHO to have the envoy::api::v2:: prefix in the type, indicating that this is a proto.

Reply:
+1